pkg/operator/podsecurityreadinesscontroller/conditions.go

```diff
@@ -0,0 +1,89 @@
+package podsecurityreadinesscontroller
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	ViolationReason = "PodSecurityReadinessController"
+	ExpectedReason  = "AsExpected"
+
+	PodSecurityCustomerType     = "PodSecurityCustomerEvaluationConditionsDetected"
+	PodSecurityOpenshiftType    = "PodSecurityOpenshiftEvaluationConditionsDetected"
+	PodSecurityRunLevelZeroType = "PodSecurityRunLevelZeroEvaluationConditionsDetected"
+)
+
+var (
+	// run-level zero namespaces, shouldn't avoid openshift namespaces
+	runLevelZeroNamespaces = map[string]struct{}{
+		"default":     empty,
+		"kube-system": empty,
+		"kube-public": empty,
+	}
+	empty = struct{}{}
```

Comment:
`sets.New[string]()`

Reply:
`struct{}{}` is as lightweight as possible. I can go with an empty `sets.Set[string]`, but this is less lightweight.